client/templates/home.html

```diff
@@ -1,12 +1,32 @@
 <template name="home">
 
-  {{#if currentUser}}
-    {{> meetingSeriesList}}
+  {{#if sendVerificationEmailEnabled}}
+      {{#unless currentUser.emails.[0].verified}}
+          <p class="alert alert-warning">You need to verify your email address before.
```

Comment:
Okay, I will approve the changes. I opened a new issue and described the aspects there (see #250). @derwok please check and extend the story.